functions/Get-DbaTopResourceUsage.ps1

```diff
@@ -98,10 +98,10 @@
 						    coalesce(db_name(st.dbid), db_name(cast(pa.value AS INT)), 'Resource') AS [Database],
 						    coalesce(object_name(ST.objectid, ST.dbid), '<none>') as ObjectName,
 						    qs.query_hash as QueryHash,
-						    qs.total_elapsed_time as TotalElapsedTime,
+						    qs.total_elapsed_time / 1000 as TotalElapsedTimeMs,
```

Comment:
Right, if ms is short for milliseconds, then elapsed time (in microseconds) / 1000 is indeed in milliseconds.